modules/porous_flow/src/materials/PorousFlowPermeabilityConst.C

```diff
@@ -0,0 +1,32 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowPermeabilityConst.h"
+
+template<>
+InputParameters validParams<PorousFlowPermeabilityConst>()
+{
+  InputParameters params = validParams<Material>();
+
```

Comment:
I don’t understand what you mean. I’ve got validParams?

From: Andy Wilkins [mailto:notifications@github.com]
Sent: Friday, 5 August 2016 8:18 AM
To: WilkAndy/moose <moose@noreply.github.com>
Cc: Sheldon, Heather (Mineral Resources, Kensington WA) <Heather.Sheldon@csiro.au>; Author <author@noreply.github.com>
Subject: Re: [WilkAndy/moose] Initial porous flow 6845 - poroperm functions (#45)


In modules/porous_flow/src/materials/PorousFlowPermeabilityConst.C<https://github.com/WilkAndy/moose/pull/45#discussion_r73625043>:

> @@ -0,0 +1,32 @@

> +/****************************************************************/

> +/* MOOSE - Multiphysics Object Oriented Simulation Environment  */

> +/*                                                              */

> +/*          All contents are licensed under LGPL V2.1           */

> +/*             See LICENSE for full restrictions                */

> +/****************************************************************/

> +

> +#include "PorousFlowPermeabilityConst.h"

> +

> +template<>

> +InputParameters validParams<PorousFlowPermeabilityConst>()

> +{

> +  InputParameters params = validParams<Material>();

> +

Extra line not needed. Also, you have to have validParams();

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/WilkAndy/moose/pull/45/files/ce366d17c08b92cdb7d58f1273857ae5c8851b49#r73625043>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKJ_8m8fNa3S5_lLrvsCrN8zDU-XyVIJks5qcoEcgaJpZM4JZPBg>.